Query/events/findings/security_finding.json

```diff
@@ -0,0 +1,9 @@
+{
+  "attributes": {
+    "devices": {
+      "caption": "Devices",
+      "description": "Devices associated with the security finding.",
+      "is_array": true,
+      "type": "device"
+	}
+}
```

Comment:
While this looks good, I am wondering why limit it to only device. We should add our primary entity types like user.